assets/worker/0700_worker_daemonset.yaml

```diff
@@ -13,8 +13,13 @@ spec:
       labels:
         app: nfd-worker
     spec:
```

Comment:
```suggestion
    spec:
      tolerations:
      - operator: "Exists"
        effect: "NoSchedule"

```

Reply:
Tested this patch and it makes the operator to run in a non desired state.

Reply:
to be clear, you propose adding these lines without removing the other additions ?

Reply:
If I add those lines,  both `nfd-master` and `nfd-worker` pods start shutting on and off 
```bash
NAME                            READY   STATUS        RESTARTS   AGE
nfd-master-22qbg                1/1     Terminating   0          23s
nfd-master-pfcgq                1/1     Running       0          2m39s
nfd-master-vt66q                1/1     Running       0          2m39s
nfd-operator-5d9955799d-npmx7   1/1     Running       0          2m41s
nfd-worker-dnxg7                1/1     Running       0          2m39s
nfd-worker-dwfpj                1/1     Running       0          2m39s
nfd-worker-tphv4                1/1     Terminating   0          81s
```

they go from Running to Terminating and back in a loop

Reply:
You have some other problem with your setup. Running two operators or smth? How would worker spec affect master??

And yes, add both tolerations and affinity. Affinity does not make any sense without the toleration. Agree?